src/pages/patientView/SampleManager.tsx

```diff
@@ -115,6 +84,45 @@ export function sortSamples(
     });
 }
 
+export function getSpecimenCollectionDayMap(
+    sampleIDs: string[],
+    events?: ClinicalEvent[]
+) {
+    let collectionDayMap = new Map<string, number>();
+    if (events) {
+        // use SPECIMEN or SAMPLE_ACQUISITION track on timeline to get timeline
+        // event
+        // TODO: SAMPLE_ACQUISITION is specific to genie_bpc_test study. We
+        // should probably have some config to allow people to choose what
+        // timeline tracks get labels
+        const specimenEvents = events.filter(
+            (e: ClinicalEvent) =>
+                e.eventType === 'SPECIMEN' || 'SAMPLE_ACQUISITION'
+        );
+
+        specimenEvents.forEach((event: ClinicalEvent) => {
+            let sampleID = _.find(
```

Comment:
I would avoid using name sampleID for a object.  The ID suffix strongly suggests string.  And we use the name sampleId everywhere.  Maybe sampleObj?

Reply:
Will do!